### PR TITLE
Added level 4 and 5

### PR DESCRIPTION
**Pull request by vzorglub**

### CODE REVIEW

**Comment by vzorglub** [comment]
Hi,
New to git hub
There was an error in the first file I tried to commit
I corrected the error and the committed but it appears that both files have been committed
I also don't know how to change version number and all that in the main presentation files
I hope you can make sense of it

**Comment by rogeriopvl** [comment]
Hi @vzorglub thanks! Awesome job on your first pull request. I'll be reviewing as soon as possible.

File: windrose.js [hunk 1]
```diff
@@ -32,40 +32,136 @@
...
         { symbol: 'NW', name: 'North West', depth: 1 },
+        { symbol: 'NW¼N', name: 'North West quarter North', depth: 5 },
+        { symbol: 'NW½N', name: 'North West half North', depth: 4 },
+        { symbol: 'NW¾N', name: 'North West three quarters North', depth: 5 },
         { symbol: 'NWbN', name: 'North West by North', depth: 3 },
+        { symbol: 'NNW¾W', name: 'North North West three quarters West', depth: 5 },
+        { symbol: 'NNW½W', name: 'North North West half West', depth: 4 },
+        { symbol: 'NNW¼W', name: 'North North West quarter West', depth: 5 },
         { symbol: 'NNW', name: 'North North West', depth: 2 },
-        { symbol: 'NbW', name: 'North by West', depth: 3 }
```
Comment (rogeriopvl) [inline windrose.js]: Why is this being removed?

Comment (rogeriopvl) [reply windrose.js]: Ok, nevermind. I got it 👍

**Comment by rogeriopvl** [comment]
Closed by #8

**Pull request closed without merging**